extensions/processing/README.md

```diff
@@ -0,0 +1,64 @@
+# Processing Extension Specification
+
+- **Title: Processing**
+- **Identifier: proc**
+- **Field Name Prefix: proc**
+- **Scope: Item, Collection**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+Processing metadata is considered to be data that indicate from which processing chain a data originates and how the data itself has been produced. Overall, it helps to increase traceability and search among processing levels and multiple algorithm versions.
+
+Often, data items are the result of one or more waterfall processing pipeline. Tracing information such as the processing facility , the algorithm version or the processing date helps in the data version management.
+
+This extension applies to STAC Items and STAC Collections. As these processing information are often closely bound to the collection level and therefore are shared across all items, it is recommended adding the fields to the corresponding STAC Collection.
+
+- [Examples](examples/): [Sentinel-1 Item](examples/sentinel1-item.json),  [Sentinel-2 Level 1C Collection](examples/sentinel2-collection.json)
+- [JSON Schema](json-schema/schema.json)
+
+## Item and Collection fields
+
+For Items, the fields are placed in the properties. For Collections, the fields are placed in the summaries of the Collection.
+
+| Field Name                  | Type      | Description                                                  |
+| --------------------------- | --------- | ------------------------------------------------------------ |
+| proc:name         | string | The name of the processing of the data that is commonly used. For example, `GRD Post Processing` for "GRD" product of Sentinel-1 satellites. |
+| proc:level        | string | **REQUIRED.** The name commonly used to refer to the processing level to make it easier to search for product level across collections or items. The short name must be used (only `L`, not `Level`). See the [list of suggested processing levels](#suggested-processing-levels). |
+| proc:facility     | string    | The name of the facility that produced the data. For example, `Copernicus S1 Core Ground Segment - DPA` for product of Sentinel-1 satellites. |
+| proc:software     | string    | The name of the software that produced the data. For example, `Sentinel-1 IPF` for the software that produces Sentinel-1 satellites data. |
+| proc:version      | string | The version of the software that produced the data. For example, `002.71` for the software that produces Sentinel-1 satellites data. |
```

Comment:
I think I'd prefer proc:software_version to clarify the scope of the field.

Also, I'm wondering whether multiple software could be required? Then it might make sense to do something like:

```
proc:software: {
  "Sentinel-1 IPF": "002.71",
  "Other Tool": "1.1.1"
}
``` 
or something similar (could also be an array with objects).

Reply:
Fine with the pair software "name:version"

Reply:
+1 to software_version. I'm neutral on string vs array/object